src/main/java/org/fundacion/jala/converter/view/compiler/ProjectTab.java

```diff
@@ -79,10 +82,53 @@ public boolean removeTap(String title) {
     }
 
     /**
-     * It represents what CodeTextArea is selected
-     * @return a CodeTextArea
+     * It represents what CodeTextArea is selected.
```

Comment:
It may be better without the "what" and the "It" on the description. Just say: "Represents the selected CodeTextArea"